srcpy/py_tricamera_types.cpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file
+ * @brief Create bindings for three pylon dependent camera sensors
+ * @copyright 2020, New York University, Max Planck Gesellschaft. All rights
+ *            reserved.
+ * @license BSD 3-clause
+ */
+
+#include <robot_interfaces/sensors/tricamera_observation.hpp>
+#ifdef Pylon_FOUND
+#include <robot_interfaces/sensors/tricamera_driver.hpp>
+#endif
+
+#include <robot_interfaces/sensors/pybind_sensors.hpp>
+#include <robot_interfaces/sensors/sensor_driver.hpp>
+
+using namespace robot_interfaces;
+
+PYBIND11_MODULE(py_tricamera_types, m)
+{
+    create_sensor_bindings<TriCameraObservation>(m);
+
+#ifdef Pylon_FOUND
+    pybind11::class_<TriCameraDriver,
+                     std::shared_ptr<TriCameraDriver>,
+                     SensorDriver<TricameraObservation>>(m, "TriCameraDriver")
```

Comment:
It is still with lower-case "c" here.  Did you test compiling on a machine with Pylon installed?

Reply:
Sorry about that, I had made a mistake in testing. Fixed now. Thanks for pointing this out.